apps/base-docs/server.js

```diff
@@ -430,6 +430,9 @@ const contentSecurityPolicy = {
     'https://cdn.sprig.com', // Sprig API
     'https://flag.lab.amplitude.com/sdk/v2/flags',
     'https://api.lab.amplitude.com/sdk/v2/vardata',
+    'https://browser-intake-datadoghq.com', // datadog
+    'https://app.datadoghq.com', // datadog
+    'datadoghq.com' // datadog
```

Comment:
```suggestion
    'https://*.datadoghq.com', // datadog
```

I think you can wildcard this to get https enforcement and auth all subdomains